test/activities/content/data/TestContentModuleEntity.js

```diff
@@ -39,10 +39,17 @@ export const contentModuleData = {
 	'properties': {
 		'title': 'Test Content Module Title'
 	},
-	'entities': [{
-		'class': ['richtext', 'description'],
-		'properties': { 'text': 'description text', 'html': '<p>description text</p>' },
-		'rel': []
-	}],
+	'entities': [
+		{
+			'class': ['richtext', 'description'],
+			'properties': { 'text': 'description text', 'html': '<p>description text</p>' },
+			'rel': []
+		},
+		{
+			'class': ['richtext', 'raw-description'],
+			'properties': { 'text': 'description text', 'html': '<p>description text</p>' },
+			'rel': []
+		}
```

Comment:
As for the comment above, we have serialized both the raw and the useable description so having that  distinction on the activities side makes sense (unless I completely misunderstood the PR)

Reply:
I was unsure if the description entity is only consumed by us, if it is used elsewhere I did not want to change it. Also, the string replacement [occurs here](https://github.com/Brightspace/lms/blob/6c6462fec92e911f26701f1bb6eec3238945c8bd/lp/framework/core/D2L/LP/TextProcessing/Rendering/Renderers/UserEnteredHtmlHtmlRenderer.cs#L53). There are a bunch of text renderers and it looks like this is the one that is used by FACE to populate the editor.